Pipfile

```diff
@@ -30,3 +30,4 @@ coverage = "*"
 coveralls = "*"
 pytest-cov = "*"
 jupyterlab = "*"
+mock = "*"
```

Comment:
mock is part of the standard library right? this might not be necessary

Reply:
We need to install it. I got an error when I didn't install it.

Reply:
Ahhh interesting. It's under `unittest.mock` starting with version 3.3 but they keep a version on PyPI for backwards compatibility.   

https://mock.readthedocs.io/en/latest/